apps/spruce/src/pages/waterfall/WaterfallGrid.tsx

```diff
@@ -159,13 +161,18 @@ export const WaterfallGrid: React.FC<WaterfallGridProps> = ({
 
 const Container = styled.div``;
 
-const StickyHeader = styled(Row)`
+const StickyHeader = styled(Row)<{ atTop: boolean }>`
   position: sticky;
   top: -${size.m};
   z-index: 1;
 
   background: white;
   padding: ${size.xs} 0;
```

Comment:
If you want to extend the border all the way to the edge it seems like this works
```suggestion
  margin-left: -${size.m};
  margin-right: -${size.m};
  padding: ${size.xs} ${size.m;
```

Reply:
omg yay!